misc/nuitka-package-config-schema.json

```diff
@@ -105,10 +100,9 @@
                             }
                         },
                         "control_tags": {
-                            "type": "array",
-                            "items": {
-                                "type": "string"
-                            }
+                            "type": "string",
+                            "pattern": "(([a-zA-Z]|[0-9]|_)|and|is|not| )"
```

Comment:
Does the schema format allow to define this and use it later somehow, this is repeating itself a lot otherwise.

Reply:
I think I can do that.